serving-sdk/src/main/java/org/pytorch/serve/servingsdk/metrics/DimensionRegistry.java

```diff
@@ -0,0 +1,18 @@
+package org.pytorch.serve.servingsdk.metrics;
+
+/**
+ * This is a registry for different metric dimensions and their values.
+ * This is not exhaustive list. Dimension can be added to Metric without adding it to this Registry.
+ */
+public class DimensionRegistry {
```

Comment:
NIT: Not quite clear where `DimensionRegistry` and `InbuiltMetricsRegistry` would be used

Reply:
The purpose of these registries is to list and refer all the Metrics/Dimensions available in TorchServe in one place. It is expected that whenever a new metric/dimension gets added in TorchServe, a corresponding entry should be made here. The Plugin writers then should be able to refer to this list, and will not have to worry about the spelling of metric, duplicate metric name, etc. However, currently, it is not mandatory to make an entry here.  I have updated the Doc String to reflect this comment.